cpp/libcudf_kafka/CMakeLists.txt

```diff
@@ -94,7 +96,7 @@ install(
   EXPORT cudf_kafka-exports
 )
 
-install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include DESTINATION include)
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
```

Comment:
This is a pretty subtle change but fixes a major issue with packaging. Without this slash, `libcudf_kafka` has been shipping its headers in `include/include/cudf_kafka/...` which led to bad practices in the Python packaging for finding those headers.